app/views/admin/submissions/_created_by.html.erb

```diff
@@ -6,7 +6,7 @@
     <% if @submission.admin %>
       <%= mail_to @submission.admin.email, @submission.admin.email, class: "smaller-sidebar-link" %>
     <% elsif @submission.user&.gravity_user_id.present? %>
-      <%= link_to @submission.user&.gravity_user_id, user_management_url(@submission.user&.gravity_user_id), class: "smaller-sidebar-link" %>
+      <%= link_to @submission.name.presence || @submission.user&.gravity_user_id, user_management_url(@submission.user&.gravity_user_id), class: "smaller-sidebar-link" %>
```

Comment:
just noticed small difference, in another file you do `submission.name` without `presence`

Reply:
Ahh, thanks — I'll update both to use `#presence` as that should catch any strings with only whitespace.